lib/onelogin/ruby-saml/authrequest.rb

```diff
@@ -28,6 +28,8 @@ def create(settings, params = {})
 
       def create_params(settings, params={})
         params = {} if params.nil?
+        # Some ruby-saml versions uses :RelayState others use 'RelayState'
```

Comment:
It might be that some people are using `HashWithIndifferentAccess` when they use this gem in Rails.